components/tikv_util/src/config.rs

```diff
@@ -677,6 +677,38 @@ pub fn check_max_open_fds(expect: u64) -> Result<(), ConfigError> {
     }
 }
 
+#[cfg(target_os = "freebsd")]
+pub fn check_max_open_fds(expect: i64) -> Result<(), ConfigError> {
```

Comment:
Nit: here is the minimum change.

```suggestion
#[cfg(unix)]
pub fn check_max_open_fds(expect: u64) -> Result<(), ConfigError> {
    #[cfg(target_os = "freebsd")]
    let expect = expect as i64;
```

Reply:
Thanks, I've updated the PR